tests/Doctrine/Tests/ORM/Persisters/JoinedSubclassPersisterTest.php

```diff
@@ -39,6 +39,6 @@ protected function setUp()
      */
     public function testExecuteInsertsWillReturnEmptySetWithNoQueuedInserts()
     {
-        $this->assertSame([], $this->persister->executeInserts());
+        $this->assertEmpty($this->persister->executeInserts());
```

Comment:
We cannot do this yet since `EntityPersister#executeInserts()` has no return type declaration, so we must ensure that we have the correct type.